src/helpers/index.tsx

```diff
@@ -0,0 +1 @@
+export const sleep = (delay: number) => new Promise((resolve) => setTimeout(resolve, delay));
```

Comment:
Because of how we're using this in Storybook examples I think it would be good to keep this helper localised to there and not available in the main UI Kit exports. Mostly because I'm worried that it might get misused in production code. Not a high chance but still a chance!

Reply:
Understood, I will move it to be just on Storybook.